index.js

```diff
@@ -378,11 +378,27 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = null
+    var versionMsg = null
+
+    if (arguments.length === 1) {
+      version = opt
+      versionOpt = 'version'
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else if (arguments.length === 2) {
+      versionOpt = opt
+      version = msg
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else {
```

Comment:
@nexdrew I actually was thinking about accepting an object as well. I think, accepting strings via the snowball effect (with support for `null`) <i>or</i> an object would be the easiest to code, yet the easiest to use as an API, what do you think?